src/webots/vrml/WbNode.hpp

```diff
@@ -126,9 +126,9 @@ class WbNode : public QObject {
 
   // error reporting
   // field name will be extracted from the first single quoted text in the message
-  void warn(const QString &message) const;  // show standard warning message formatted for this node
-  void info(const QString &message) const;  // show standard info message formatted for this node
-  QString usefulName() const;               // user friendy node name for error messages
+  void warn(const QString &message, bool parsingMessage = true) const;  // show standard warning message formatted for this node
+  void info(const QString &message, bool parsingMessage = true) const;  // show standard info message formatted for this node
```

Comment:
It seems dangerous to use a default value to true for the parsing case. Maybe next time we will write a warning or an info for a node, we will likely forget to set this flag to false and it will appear as a parsing message. I would reverse this and set the default value to false which seems safer to me...

Reply:
I don't see why the other way would be safer, it will just reverse the problem:
Next time we will write a warning or an info for a node we will likely forget to set this flag to true and it will appear as an normal message and not as a parsing message.

To me the default argument in that case should reflect the most used case, which in that case is quite clear (just a few tens for non-parsing messages VS ~500 for the parsing case).

If we really want to be safe then we should just make this argument mandatory.